src/main/scala/com/ebiznext/comet/utils/Job.scala

```diff
@@ -64,10 +65,32 @@ trait JobBase extends StrictLogging with DatasetLogging {
 
 trait SparkJob extends JobBase {
 
-  def withExtraSparkConf(): Map[String, String] = Map.empty
+  def withExtraSparkConf(sourceConfig: SparkConf): SparkConf = {
+    // During Job execution, schema update are done on the table before data is written
+    // These two options below are thus disabled.
+    // We disable them because even though the suer asked for WRITE_APPEND
+    // On merge, we write in WRITE_TRUNCATE mode.
+    sourceConfig.remove("spark.datasource.bigquery.allowFieldAddition")
+    sourceConfig.remove("spark.datasource.bigquery.allowFieldRelaxation")
+    sourceConfig
+  }
```

Comment:
IMO this should be in BigQueryJobBase, not here.
This isn't related to Parquet2CSV, or MetricsJob, or ESLoadJob